TicketSystem/TicketSystem.Tests/IntegrationTests/ControllersTests/TicketControllerTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.AspNetCore.Mvc.Testing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.DependencyInjection;
+using TicketSystem.DAL;
+using TicketSystem.Tests.Initialize;
+using Newtonsoft.Json;
+using Shouldly;
+using TicketSystem.API.ViewModels.Messages;
+using TicketSystem.API.ViewModels.Tickets;
+using TicketSystem.API;
+using System.Net;
+using System.Net.Http.Json;
+using TicketSystem.Tests.IntegrationTests.WebAppFactory;
+
+namespace TicketSystem.Tests.IntegrationTests.ControllersTests
+{
+    public class TicketControllerTests : IClassFixture<TestHttpClientFactory<Program>>
+    {
+        private readonly HttpClient _httpClient;
+        private const int OpenTicketId = 1;
+
+        public TicketControllerTests(TestHttpClientFactory<Program> factory)
+        {
+            _httpClient = factory.CreateClient();
+        }
+
+        [Fact]
+        public async Task GetAllTickets_ReturnIEnumerableTicketViewModels()
+        {
+            // Act
+            var response = await _httpClient.GetAsync("/api/ticket");
+
+            // Assert
+            response.EnsureSuccessStatusCode();
+            var ticketsViewModel = await response.Content.ReadFromJsonAsync<IEnumerable<TicketViewModel>>();
```

Comment:
Perhaps, it is better to move in "Act" part

Reply:
see comment above